travvy/src/components/AttractionsList.vue

```diff
@@ -118,13 +105,82 @@
+        <h3>Thomas, from Denver, Colorado: ⭐⭐⭐⭐</h3>
+        <p>Great city with diverse culture and great education and jobs if you want to be successful.Also every one is helpful and people are great.</p>
+        <h3>Laura, from Burlington, Ontario: ⭐⭐⭐</h3>
+        <p>There are nice food places in New York that offer a variety of food choices such as pizza and the classic bacon, egg, and cheese. However, the city is very crowded and I found myself getting lost.</p> 
+      </div>
+      <div v-else-if="$store.state.city == 'Rio de Janeiro'">
+        <h3>Steven, from Montreal, Quebec: ⭐⭐⭐⭐⭐</h3>
+        <p>Cariocas, as natives of Rio call themselves, refer to their hometown as "The Marvelous City," and this is no exaggeration. In the beach, it might be compared to Miami, Honolulu, Monte Carlo or Acapulco, but Rio is better. A week there was not enough.</p>
+        <h3>Chris, from Los Angeles, California: ⭐⭐⭐⭐⭐</h3>
+        <p>Rio was amazing! Just stay safe buy trying to blend in, and leaving your jewelry at home. As long as you don't walk around with anything flashy you will be fine. Locals dont wear anything flashy so that makes you stand out. Be smart and you will have the best experience.</p>
+        <h3>Sean, from Venlo, Netherlands: ⭐⭐⭐</h3>
+        <p>Disappointed. But as with all the ‘famous’ spots in a city the truth is not as advertised. Built up, nosy and the beach does not seem or feel as clean as nearby beaches.</p> 
+      </div>
+      <div v-else-if="$store.state.city == 'Rome'">
+        <h3>Alex, from Berlin, Germany: ⭐⭐⭐⭐⭐</h3>
+        <p>The Colosseum is also very very must go when you are in Italy. This chance, i went there to have a pictures and even without a good angle - its a good picture already.</p>
+        <h3>Danny, from Mumbai, India: ⭐⭐⭐⭐</h3>
+        <p>Rome fulfils all that it promises. History, paintings, sculptures and food. The Vatican, the Colosseum,the Trevi fountain and lots more. Ideally a month would be less to savour all these things in some detail. </p>
+        <h3>Yonina, from Sofia, Bulgaria: ⭐⭐⭐⭐</h3>
+        <p>For nearly free, we covered all the major sites in the Roman forum with a fantastic guide who really gave us an excellent overview.</p> 
+      </div>
+      <div v-else>
+        <!--handling of empty recommended attractions -->
+          <h1> No reviews available </h1>
+      </div>
+
+      <!--<div v-if="$store.state.recommendedAttractions.length > 0">
+      <p>{{$store.state.recommendedAttractions[0].city}} felt pretty safe but if you are worried about COVID it may not be the best time to go, it is tough to get around without being around a lot of people.</p>
+      <h3>Lisa, 28 from Winnipeg, ⭐ ⭐ ⭐ ⭐</h3>
+      <p>I spent time at {{$store.state.recommendedAttractions[0].attraction_name}}, I'd totally recommend it. Great view and less people.</p>
+      </div>
+      <div v-else>
+       </div>-->
+      
   </div>
 </template>
 
```

Comment:
Looks great Lucas! Maybe in a future sprint we could look into how this information could retrieved through an API!